.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Issue Details
+(closes, fixes, suggestion) for #issue-number 
+
+## Describe the changes proposed
+For eg.
+1. This PR makes the login form responsive for small screen.
```

Comment:
Instead of giving a example, let's add a few  pointers instead. Otherwise it could happen that the template example, and the actual contribution are the same which might confuse maintainers.

Let's put some bullet points similar to this

```
# What changes does your PR create?
# Have you added or deleted something?
# How might this affect the overall project?
```

Reply:
Yes . example can be confusing . bullet 1 and 2 look pretty similar hence adding only 1 and 3.